src/components/TrackList.demo.js

```diff
@@ -16,6 +16,7 @@ export default (
             mainPalette: "blues",
             auxPalette: "reds",
             colorReadsByMappingQuality: false,
+            alphaReadsByMappingQuality: false,
```

Comment:
If we're adding a new key to the color settings type we should update the schema we have here: https://github.com/vgteam/sequenceTubeMap/blob/0452ecb82d057372e359a9b456d789336e5ab8a1/src/Types.ts#L85